app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<% provide :page_title, construct_page_title(t('.title')) %>
+
+<h1><%= t '.header' %></h1>
```

Comment:
In HydraNorth Login page we were using an H2? Should be an H1. 

Also I am i18n everything where possible (some stuff from that comes out of sufia is not really possible to get from i18n).